worldedit-sponge/build.gradle

```diff
@@ -0,0 +1,99 @@
+apply plugin: 'eclipse'
```

Comment:
I really don't like working with gradle build scripts all that much, so if you can think of a nice way to do it I won't object. Although, it might be nice to keep them separate that way the Sponge and Forge versions of WorldEdit can differ.